src/alarms/notifications/AlarmRow.js

```diff
@@ -13,64 +13,6 @@ class AlarmRow extends Component {
       isFetching: true,
       isActive: props.alarm.active
     };
-    this.activateAlarm = this.activateAlarm.bind(this);
-    this.deActivateAlarm = this.deActivateAlarm.bind(this);
-  }
-
```

Comment:
Deze functies heb ik allemaal naar het bovenliggende component app.js verplaatst (component dat de lijst toont).

Niet helemaal zeker of dit de juiste keuze is. Deze keuze zorgt er in ieder geval voor dat het api fetch gedeelte binnen de ap.js lijst component blijft.